prep-html-relative-pathing/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+<div><a href="/index.html">Click here to go to the home page</a></div>
```

Comment:
This is pathing from the root of your application. Please make use of a relative path, which starts with either `../` or the file or folder name you want to access. Let me know if you have any questions!